Framework/src/ObjectsManager.cxx

```diff
@@ -35,6 +35,11 @@ ObjectsManager::~ObjectsManager()
 
 void ObjectsManager::startPublishing(TObject* object)
 {
+  if (mMonitorObjects.FindObject(object->GetName()) != 0) {
+    QcInfoLogger::GetInstance() << "Object already published (" << object->GetName() << "), we won't republish"
```

Comment:
Maybe 'being published' instead of 'published'? I think this might be interpreted as "Object have been already published", though my understanding might be wrong here.

Reply:
correct, fixing it now.